app/Models/BeatmapDiscussion.php

```diff
@@ -659,6 +659,7 @@ public function restore($restoredBy)
                 BeatmapsetEvent::log(BeatmapsetEvent::DISCUSSION_RESTORE, $restoredBy, $this)->saveOrExplode();
             }
 
+            $this->beatmapDiscussionPosts()->where('deleted_at', $this->deleted_at)->update(['deleted_at' => null]);
```

Comment:
this means post previously deleted separately also get restored, right?

Reply:
the where should exclude those out 🤔

Reply:
I did not see that 👀